src/app/navbar.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import styles from "./Navbar.module.css";
+
+export default function Navbar() {
+  const pathname = usePathname();
+
+  const pageLinksData = [
+    {
+      name: "Home",
+      url: "/",
+    },
+    {
+      name: "About",
+      url: "/about",
+    },
+  ];
+
+  const socialLinksData = [
+    {
+      name: "Discord",
+      url: "https://discord.gg/5SyB3yx",
+      icon: "/discord_mark.svg",
```

Comment:
[Nit]: These should probably all be called `x_logo.svg` or something rather than separate names, I'm not sure what's meant by `mark` vs `logo` vs `icon` vs `glyph`. Assuming some are `png`s as an `svg` couldn't be found; that's fine.